build.sbt

```diff
@@ -10,6 +10,7 @@ name := "sbt-license-report"
 enablePlugins(SbtPlugin)
 libraryDependencies += lang3
 scriptedLaunchOpts += s"-Dplugin.version=${version.value}"
+pluginCrossBuild / sbtVersion := "1.9.5"
```

Comment:
I am just not sure if it's worth forcing the users to use latest sbt 1.9.6 for one plugin, if they want this plugin.

Reply:
Fair point, I guess you could use reflection to see if the method already exists and if not default to the one from 1.9.5 however this is over-engineering is even more complicated than the current implementation.

I think its worth it to leave this PR open as there many be some future sbt change which is critical to this plugin at which point we can merge this PR.

On a similar note, there have been various security issues that have been fixed in more recent versions of sbt so I don't know if thats a strong enough argument to merge the PR since most users should be using a more recent version of sbt but I will leave this to your discretion.